src/types/positions.ts

```diff
@@ -95,6 +95,15 @@ export interface EarnDataProps {
   // We'll add more fields here as needed
 }
 
+interface ShortcutTriggerArgsContext {
+  tokenInfo: TokenInfo
```

Comment:
@jeanregisser unlike data / displayProps, we can't use `resolvedTokensById` here because we need the info of the token we're currently resolving, and this is contained in this `tokenInfo` field. But this applies only for app tokens and not contract positions, any preferences on how to deal with this for contract positions?

Reply:
oh I see, I guess we could pass `tokensByTokenId` instead, which includes all app tokens info?